src/main/java/sirius/biz/tenants/mongo/MongoTenantAutoSetup.java

```diff
@@ -45,24 +45,27 @@ public void setup() throws Exception {
         }
 
         AutoSetup.LOG.INFO("Creating system tenant....");
```

Comment:
should we adjust the logic in SQLTenantAutoSetup to match the following?

Reply:
done